MTM-64077 - release notes - Improved index utilization for time series measurements (#3764)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-2025-335-0-time-series-measurements-index-improvements.md

---
date: 
title: Improved index utilization for time series measurements
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
product_area: Platform services
component:
  - value: component-JlFdtOPva
    label: REST API
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-64077
version: 2025.335.0
---

> Index utilization for time series measurements has been improved. The platform uses database indexes more efficiently when storing and

Reviewer: ```suggestion
The platform now uses database indexes more efficiently when storing and
```
